src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -317,21 +317,29 @@ public void TestMailDelivery()
         }
 
         [Fact]
-        [ActiveIssue(40711)]
+        // [ActiveIssue(40711)]
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework has a bug and may not time out for low values")]
         [PlatformSpecific(~TestPlatforms.OSX)] // on OSX, not all synchronous operations (e.g. connect) can be aborted by closing the socket.
         public void TestZeroTimeout()
         {
-            using (Socket serverSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
+            var testTask = Task.Run(() =>
```

Comment:
nit: 'var' shouldn't be used here. It's only used when the type is obvious (from a `new` or a cast).